sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -928,6 +931,22 @@ def _mapping_event_from_change_event(self, status, port_dict):
         helper_logger.log_debug("mapping from {} {} to {}".format(status, port_dict, event))
         return event
 
+    # Soak SFP insert event until management init completes
+    def _soak_sfp_insert_event(self, port_dict):
+        self.sfp_insert_events = {}
```

Comment:
can you please check get_change_event for all the vendors?. Incase empty port dict is returned, timeout will not work properly. In this case, do we have to re-initialize sfp_insert_events?.

Reply:
insertion time is updated only once when module is inserted (the event is then deleted from port_dict)